frontend/common/src/lang/fr.json

```diff
@@ -107,11 +107,11 @@
     "description": "Label for the privacy link in the Footer."
   },
   "ZE77nf": {
-    "defaultMessage": "canada.ca",
+    "defaultMessage": "Canada.ca",
     "description": "Label for the Canada link in the Footer."
   },
   "ZGpncy": {
-    "defaultMessage": "Modalités",
+    "defaultMessage": "Avis",
```

Comment:
Changing this means that an e2e fails, so will need to [update](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/cypress/integration/talentsearch/footer.spec.js#L41) that as well as some other examples.